ManagerRegistry.php

```diff
@@ -20,12 +20,38 @@
 
 namespace Doctrine\Bundle\PHPCRBundle;
 
+use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Bridge\Doctrine\ManagerRegistry as BaseManagerRegistry;
 use Doctrine\ODM\PHPCR\PHPCRException;
 
 class ManagerRegistry extends BaseManagerRegistry
 {
     /**
+     * @var ContainerInterface
+     */
+    private $container;
```

Comment:
The `ManagerRegistry` class of DoctrineBridge already defines this field!

Reply:
Ok. Thanks